.github/CODEOWNERS

```diff
@@ -3,3 +3,4 @@
 # For more info, see https://help.github.com/articles/about-codeowners/
 
 * @elastic/obs-docs
+/.github/workflows/ @elastic/docs-engineering
```

Comment:
>I'm fine with @elastic/docs-engineering owning the build-specific workflows, but I'm not sure it makes sense for that team to own (and have to review) all of our workflows.

Yea. The specific workflow they need permission for hasn't been added to this repo yet. Katie is doing that next. We can remove this until that is done. 

```suggestion
```